wideq.py

```diff
@@ -679,6 +679,40 @@ def enum_name(self, key, value):
         options = self.value(key).options
         return options[value]
 
+    @property
+    def monitor_type(self):
+        """Get type of monitoring return data.
```

Comment:
Perhaps the docstring could mention what 1 and 0 mean as the values here? I suppose, alternatively, the property could just be a Boolean (called something like `binary_monitor_data`).